src/components/Calendar/calendar.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,500,500i,700,700i,900,900i');
 .events {
-    max-height: 300px;
+    height: 300px;
```

Comment:
This should be reverted to 'max-height: 300px'. This was a fix for a display issue on the calendar page and should not have any effect on tests.